Windows/MainWindow.cpp

```diff
@@ -880,9 +880,10 @@ namespace MainWindow
 			break;
 
 		case WM_MENUSELECT:
-			// Unfortunately, accelerate keys (hotkeys) shares the same enabled/disabled states
```

Comment:
Yeah, you're absolutely right. I was in a rush since I had to leave, and apparently such a rush that I didn't even read the comment :(

An easier and safer approach than mucking about with frame flags might be to add a flag field to UpdateUI to choose whether to reload the translations (which is the other use for it) or to just enable/disable items. The latter might be fast enough that we don't need to care.